tests/AutoReview/ComposerTest.php

```diff
@@ -33,7 +33,7 @@ public function testBranchAlias(): void
         $composerJson = json_decode(file_get_contents(__DIR__.'/../../composer.json'), true);
 
         if (!isset($composerJson['extra']['branch-alias'])) {
-            $this->addToAssertionCount(1); // composer.json doesn't contain branch alias, all good!
+            $this->expectNotToPerformAssertions(); // composer.json doesn't contain branch alias, all good!
```

Comment:
I think this whole test method can go as we have no `branch-alias` in `composer.json` anymore

Reply:
No only this file: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/6200

Reply:
nice catch :+1:  merged :)